ansible/master_cfg/letsencrypt.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# TODO: Change the email to something that makes more sense
```

Comment:
Maybe use openscap-maint@redhat.com? But there it could get easily lost because of lots of automated emails from Jenkins.